drivers/storage/portworx/preflight.go

```diff
@@ -369,6 +380,13 @@ func (u *preFlightPortworx) processNodesChecks(recorder record.EventRecorder, st
 		}
 	}
 
+	if isFatal { // Add events for fatal checks
```

Comment:
curious why this block of code cannot be moved in the `if` block for line 366.

Reply:
The loop will process checks looking for any failures.  However with this PR there is a difference between a failure and "fatal" failure.    So I wanted to make sure the fatal events are printed after all the other failed events.     So I collected the fatal msgs so they can be sent all at once instead of being intermixed with other failed events.

Reply:
that makes sense! this definitely worth a comment

Reply:
@jrivera-px Can we have a `Fatal: ` prefix for the fatal errors and remove the `failed: ` prefix from the non-fatal errors. The reason already says "PreflightFailed".